st2api/tests/unit/controllers/v1/test_base.py

```diff
@@ -22,7 +22,7 @@ def test_defaults(self):
         response = self.app.get('/')
         self.assertEqual(response.status_int, 200)
         self.assertEqual(response.headers['Access-Control-Allow-Origin'],
-                         'http://localhost')
+                         'http://127.0.0.1:3000')
```

Comment:
This is single place where I changed the logic to make tests happy.
Verify if that's still correct.